tracer/src/Datadog.Trace/Configuration/ConfigurationSources/JsonConfigurationSource.cs

```diff
@@ -57,6 +63,8 @@ private protected JsonConfigurationSource(string json, ConfigurationOrigins orig
             _origin = origin;
         }
 
+        internal string JsonConfigurationFilePath { get; set; } = "N/A";
```

Comment:
When a property is not set, we expect the value to be `null` internally. `"N/A"` is not a valid path, just what you would display to a user if the value is not set, so that should be handle somewhere else. For example, since you're using this for log files, in the code where you add the log entry, you can use `JsonConfigurationFilePath ?? "N/A"`.

Also, since can be read-only since it's only set from the constructor, and we don't want to allow it to change afterwards.

```suggestion
        internal string JsonConfigurationFilePath { get; }
```